### ISSUE TITLE
X11 not automaticall launched, and even when launched it is used on only one X11 app.

### ISSUE THREAD

**Issue by teras** [sileshn/ArchWSL2#12]
### Requirements

- [X] Yes
- [ ] No

### CPU Architecture

- [ ] arm64
- [X] x64

### WSL Version

- [X] WSL 2
- [ ] WSL 1

### Version

Microsoft Windows [Version 10.0.22631.4169]

### Kernel Version

5.15.153.1-microsoft-standard-WSL2

### Repro Steps

All  terminals are closed. Then I open an Arch terminal. Then I try to launch an X11 app (i.e. xterm). It doesn't launch with error ```xterm: Xt error: Can't open display: :0```. 

If I close all terminals, I go back to the windows terminal and run `wsl --update` command, then when I open an arch terminal and run xterm it works. Actually it takes some more time to launch the arch terminal. 

When I open in parallel another arch terminal, still it can't connect to X11. More over, after some time (it is not deterministic I tried to find out when) the X11 "ability" disappears and a consecutive try to launch xterm fails. Only if I re-type  `wsl --update` the xterm is accessible.

### Expected Behavior

When "ubuntu" is used instead, everything works as expected. No matter how many times or how much I wait, the X11 applications run every time without any problems.

### Actual Behavior

The arch X11 applications run only once and then X11 session is vanished

**Comment by sileshn** [comment]
ArchWSL2 is not setup to run GUI ootb. Neither is it Ubuntu. So don't expect it to run like Ubuntu. 

You'll need to make changes to run GUI applications. Google the error you get or check the ManjaroWSL2 repo on instructions how to setup GUI.

**Comment by teras** [comment]
Well, I know what Arch and/or Ubuntu is. I am a hard core Linux user for almost 30 years :) Moreover I prefer to see everything as Linux, just differently packaged. So let me rephrase the question. What breaks in this Arch distribution and X server is not maintained? How can it stay alive? How can I help or where should I start to investigate this?

My motivation is to bring more people to Linux and help others, that's why I am asking. It's not for my personal benefit either (same goes to you I know). If it were just for me, I'd stay in my comfort of my Arch box :smile:

**Comment by teras** [comment]
Is there something I can do to fix this issue?
